src/SockPerf.cpp

```diff
@@ -227,6 +227,10 @@ static const AOPT_DESC  common_opt_desc[] =
 		"Limit the lifetime of the message (default 2)."
 	},
 	{
+		OPT_MC_SOURCE_IP, AOPT_ARG, aopt_set_literal( 0 ), aopt_set_string( "mc-source-ip" ),
+		"<ip, hostname> address of mulitcast messages source which is allowed to receive from."
+	},
```

Comment:
Please consider changing option description staring from letter. For example: `Set address <ip, hostname> ...`. Please change the same for *mc-rx-if* and *mc-tx-if*